src/lib/UI/Dataset/RelationsDataset.php

```diff
@@ -35,23 +36,28 @@ public function __construct(ContentService $contentService, ValueFactory $valueF
     {
         $this->contentService = $contentService;
         $this->valueFactory = $valueFactory;
+        $this->relations = [];
```

Comment:
This is not important, but if we have array as default value, I think it is more readable to set default value with properties declaration. `protected $relations = [];`

Reply:
I don't think we discussed it before. @Nattfarinn and @alongosz what is your take on it? I'm used to define default values in the constructor to keep properties section clean.

Reply:
Both ways are fine if you ask me, and not sure if it brings any value here to "standarize" it as both are clean and readable. I would not create a constructor just to assign default value, but I don't mind assigning default value if we already have some property initialisations in a constructor.